shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1975,8 +1975,11 @@ void releaseInternal() {
         @ToString.Exclude
         final String delegationToken;
         final int suggestedKeyCount;
-        final ByteBuf continuationToken; // this is used to indicate the point from which the next keys should be fetched.
-        final ByteBuf prefixFilter;      // this is used to indicate any prefix filters to apply to keys.
+        /**
+         * See {@link ReadTableEntries#fromKey}.
```

Comment:
All changes require bumping the wire protocol version.

Reply:
Done